claim/gql_mutations.py

```diff
@@ -377,6 +378,8 @@ def async_mutate(cls, user, **data):
             claim = update_or_create_claim(data, user)
             if attachments:
                 create_attachments(claim.id, attachments)
+            if is_claim_code_autogenerated:
+                return {"client_mutation_label": f"Create Claim - {claim.code}", "code": f"{claim.code}"}
```

Comment:
We need a proper way of messaging between the mutation code and the mutation handling on core. I understand this solution, but i don't like it in a long term.